### DISCUSSION TITLE
I want to make images visible

### DISCUSSION THREAD

**Discussion by xLinLord** [DirectoryLister/DirectoryLister#1051] [Help & Support] [answered]
I want to make the pictures visible in the Lister as follows
![Ekran Resmi 2023-02-19 21 08 55](https://user-images.githubusercontent.com/93214734/219972782-14256ead-d567-4cca-9c8c-48b845399cf5.png) to 
![Ekran Resmi 2023-02-19 21 09 34](https://user-images.githubusercontent.com/93214734/219972794-78630a44-04a3-46cb-9022-d56e97aceb51.png) this

**Comment by PHLAK** [comment, marked as answer]
This is not possible with Directory Lister. It sounds like you want an image gallery instead.

**Comment by breatfr** [comment]
same so i didn't use Directory Lister for my project and keep h5ai
